tests/test_experiment.py

```diff
@@ -367,3 +367,52 @@ def main(_config, nested_dict, nested_list, nested_tuple):
 
     ex.run()
 
+
+def test_add_config_dict_chain(ex):
+    # https://github.com/IDSIA/sacred/issues/409
```

Comment:
```suggestion
```
I think we do not have to refer to solved issues in the code as the test now clearly documents the intended behavior.